language-server/modules/langserver-core/src/test/resources/completion/new/expression_context/config/field_access_ctx_config1.json

```diff
@@ -249,13 +249,13 @@
       }
     },
     {
-      "label": "cloneReadOnly()(anydata)",
+      "label": "cloneReadOnly()(CloneableType)",
```

Comment:
@nadeeshaan, FYI please. Eventually this should be `Cloneable`.

Reply:
+1 to proceed. LS test changes looks good

Reply:
I will change this to `Cloneable` adding same logic added to `BUnionType` in `BallerinaUnionTypeSymbol.java`

Reply:
https://github.com/ballerina-platform/ballerina-lang/pull/27909/commits/b2c9ae1094fe2adc260d876c363edbda00b96250